kodilla-testing/src/main/java/forum/statistics/StatisticOperator.java

```diff
@@ -0,0 +1,37 @@
+package forum.statistics;
+
+public class StatisticOperator {
+    private int amountOfUsers = 0;
+    private int amountOfPosts = 0;
+    private int amountOfComments = 0;
+    private double averagePostsPerUser = 0;
+    private double averageCommentsPerUser = 0;
+    private double averageCommentsPerPost = 0;
+
+    public void calculateAdvStatistics(Statistics statistics) {
```

Comment:
Stwórz klasę `ImmutableStatistics`, w której zawrzesz te wszystkie dane, które tu - to będzie tylko tzw data-class, która tylko przechowuje dane, a nie wykonuje żadnych obliczeń.

W `calculateAdvStatistics` obliczaj wartości i stwórz obiekt klasy `ImmutableStatistics`. Następnie w testach dodaj asercje korzystając z getterów `ImmutableStatistics` by mieć pewność, że wykonuje poprawne obliczenia